stdpopsim/cli.py

```diff
@@ -750,11 +759,13 @@ def run_simulation(args):
         contig = species.get_contig(
             args.chromosome,
             genetic_map=args.genetic_map,
+            # TODO: what if we would have model.genetic_map?
```

Comment:
I don't think we need this comment here, and will cross this bridge if it comes up.

Reply:
```suggestion
```